lib/ansible/galaxy/data/collections_galaxy_meta.yml

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) 2019 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# key: The name of the key as defined in galaxy.yml
+# description: Comment/info on the key to be used as the generated doc and auto generated skeleton galaxy.yml file
+# required: Whether the key is required (default is no)
+# type: The type of value that can be set, aligns to the values in the plugin formatter
+---
+- key: namespace
+  description:
+  - The namespace of the collection.
+  - This can be a company/brand/organization or product namespace under which all content lives.
+  - May only contain alphanumeric characters and underscores. Additionally namespaces cannot start with underscores or
+    numbers and cannot contain consecutive underscores.
+  required: yes
+  type: str
+
+- key: name
+  description:
+  - The name of the collection.
+  - Has the same character restrictions as C(namespace).
+  required: yes
+  type: str
+
+- key: version
+  description:
+  - The version of the collection.
+  - Must be compatible with semantic versioning.
+  required: yes
+  type: str
+
+- key: readme
+  description:
+  - The path to the Markdown (.md) readme file.
+  - This path is relative to the root of the collection.
+  required: yes
+  type: str
+
+- key: authors
+  description:
+  - A list of the collection's content authors.
+  - Can be just the name or in the format 'Full Name <email> (url) @nicks:irc/im.site#channel'.
+  required: yes
+  type: list
+
+- key: description
+  description:
+  - A short summary description of the collection.
+  type: str
+
+- key: license
+  description:
+  - Either a single license or a list of licenses for content inside of a collection.
+  - Ansible Galaxy currently only accepts L(SPDX, https://spdx.org/licenses/) licenses
```

Comment:
This (and other links) cannot have a space between the , and the URL.  It's a flaw in the regex that parses the ```L()``` syntax.  You can fix the regex or fix the occurrences in this code of comma+space inside of an ```L()```

Reply:
If you want to fix the regex, I think this will work: ```_LINK = re.compile(r"L\(([^)]+), *([^)]+)\)")``` but, as with all regexes, it needs to be tested and examined by more than just my eyes ;-)
```

Reply:
Just opted for the simpler choice of changing the description text to not have the space.